server/index.js

```diff
@@ -1,72 +1,93 @@
-const
+const path = require('path'),
   Koa = require('koa'),
   bodyparser = require('koa-bodyparser'),
   send = require('koa-send'),
-  path = require('path'),
   staticRoot = path.join(__dirname, '../dist'),
   app = new Koa(),
-  router = require('./routes')
+  router = require('./routes');
 
-app.webpackConfig = require('../webpack.config')
+app.webpackConfig = require('../webpack.config');
 
 app.init = function(options) {
-  options = options || {}
+  options = options || {};
+
+  const useWebpack =
+    'useWebpack' in options
+      ? options.useWebpack
+      : process.env.NODE_ENV !== 'production';
+
+  let koaWebpack;
+  let compiler;
 
-  const useWebpack = 'useWebpack' in options ? options.useWebpack : process.env.NODE_ENV !== 'production'
   if (useWebpack) {
-    var Webpack = require('webpack'),
-        koaWebpack = require('koa-webpack'),
-        compiler = Webpack(app.webpackConfig)
+    const Webpack = require('webpack');
+
+    koaWebpack = require('koa-webpack');
+    compiler = Webpack(app.webpackConfig);
```

Comment:
### Manual fix:
**LN 20:** Unexpected var, use let or const instead
**LN 20 - Resolved:** Updated code to use let & const as variables are used outside scope of if statement. Should not have an impact.